public/locales/en/dial.json

```diff
@@ -9,8 +9,10 @@
     "hazardous": "Hazardous (301+)",
     "invalid": {
         "lastTime": "This sensor is not up to date. The last time it reported an accurate AQI was ",
-        "at": " at ",
+        "atPlural": " at ",
```

Comment:
It may not be worth it, but I know i18next has built in plurals support. Is this using that?

Reply:
It doesn't use that. I checked it out and it would end up being a very similar implementation to what we have rn, but if it's preferred that I do that, I can switch it.